pkg/ddevapp/ddevapp_test.go

```diff
@@ -334,6 +334,22 @@ var (
 			Type:                          nodeps.AppTypeSilverstripe,
 			DynamicURI:                    testcommon.URIWithExpect{URI: "/", Expect: "<meta name=\"generator\" content=\"Silverstripe CMS 5.0\">"},
 		},
+		// 18: CakePHP
+		{
+			Name:                          "TestPkgCakePHP",
+			SourceURL:                     "https://github.com/ddev/test-cakephp/archive/refs/tags/5.0.1.1.tar.gz",
```

Comment:
Over there, there's currently a 5.0.1.1 that is "latest" and another one with that name that's a draft. I assume delete the draft?

Reply:
Correct. I have already deleted the draft one